src/Toolbox/SAF.Toolbox/ServiceCollectionExtensions.cs

```diff
@@ -63,9 +63,18 @@ public static IServiceCollection AddRequestClient(this IServiceCollection servic
         return services;
     }
 
-    public static IServiceCollection AddFileSender(this IServiceCollection services)
+    public static IServiceCollection AddFileSender(this IServiceCollection services, IConfiguration? hostConfig = null)
```

Comment:
This is a change in the public API that breaks . To avoid major version update it might be better to add a additional method `AddFileSender` with the new parameter.

Reply:
I don't know if this is better, but I changed it. I think the default value would have preserved the compatibility.